Podfile

```diff
@@ -1,9 +1,9 @@
 use_frameworks!
 
-platform :ios, '8.0'
+platform :ios, '9.0'
```

Comment:
is this needed?

Reply:
it is needed as we only support ios 9.0+ and the pod file contains the support version.